src/components/FilterHelp/FilterHelp.jsx

```diff
@@ -1,132 +0,0 @@
-            <code>no</code> &mdash; Filters items missing certain metadata,
-            such as <code>label</code>, <code>milestone</code>, or
-            <code> assignee</code>.
-          </li>
-          <li>
-            <code>language</code> &mdash; Searches for issues or pull requests
-            within repositories that match a certain language.
-          </li>
-          <li>
-            <code>is</code> &mdash; Searches for items within repositories that
-            match a certain state, such as <code>open</code>,
-            <code> closed</code>, or <code>merged</code>
-          </li>
-          <li>
-            <code>created</code> or <code>updated</code> &mdash; Filters issues
-            or pull requests based on date of creation, or when they were last
-            updated.
-          </li>
-          <li>
-            <code>merged</code> &mdash; Filters pull requests based on the date
-            when they were merged.
-          </li>
-          <li>
-            <code>status</code> &mdash; Filters pull requests based on the
-            commit status.
-          </li>
-          <li>
-            <code>head</code> or <code>base</code> &mdash; Filters pull
-            requests based on the branch that they came from or that they are
-            modifying.
-          </li>
-          <li>
-            <code>closed</code> &mdash; Filters issues or pull requests based
-            on the date when they were closed.
-          </li>
-          <li>
-            <code>comments</code> &mdash; Filters issues or pull requests based
-            on the quantity of comments.
-          </li>
-          <li>
-            <code>user</code>, <code>org</code>, or <code>repo</code> &mdash;
-            Limits searches to a specific user, organization, or repository.
-          </li>
-          <li>
-            <code>sort</code> &mdash; Choose how to order the results. Valid
-            fields include <code>comments</code>, <code>created</code>,
-            <code> updated</code>, <code>reactions</code>,
-            <code> reactions-+1</code>, <code>reactions--1</code>,
-            <code> reactions-smile</code>, <code>reactions-thinking_face</code>,
-            <code> reactions-heart</code>, <code>reactions-tada</code>, and
-            <code> interactions</code>. Append <code>-desc</code> to sort
-            descending. Defaults to sorting newest first.
-          </li>
-          <li>
-            <code>milestone</code> &mdash; Filter by milestone name, e.g.,
-            <code> milestone:"Brilliant Features"</code>.
-          </li>
-          <li>
-            <code>reactions</code> &mdash; Filter by reaction count, e.g.,
-            <code> reactions:&gt;42</code>.
-          </li>
-          <li>
-            <code>interactions</code> &mdash; Filter by reaction + comment
-            count, e.g., <code>interactions:&lt;10</code>.
-          </li>
-        </ul>
-      </div>
-    )
-  }
-}
-
-module.exports = FilterHelp
```

Comment:
Perhaps a shorter version, with some very common ones. If you are not familiar _at all_ with the filter syntax, it's not terribly useful to have it autocomplete, because you are just staring at a blank page with no idea what to do. A short paragraph such as "Filters use standard Github formatting, for example `is:issue author:{user} sort:updated-desc`. Click here for a full list of modifiers!"